client/blocks/feedback/edit.js

```diff
@@ -9,21 +9,20 @@ import { get } from 'lodash';
  * WordPress depenencies
  */
 import { RichText } from '@wordpress/block-editor';
-import { Icon, TextControl, TextareaControl } from '@wordpress/components';
+import { Button, TextControl, TextareaControl } from '@wordpress/components';
 import { compose } from '@wordpress/compose';
 import { withSelect, dispatch } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import ConnectToCrowdsignal from 'components/connect-to-crowdsignal';
-import SignalIcon from 'components/icon/signal';
 import { withFallbackStyles } from 'components/with-fallback-styles';
 import { getFeedbackButtonPosition } from 'components/feedback/util';
 import { useAccountInfo } from 'data/hooks';
 import Sidebar from './sidebar';
 import Toolbar from './toolbar';
-import { getStyleVars, getTriggerStyles } from './util';
```

Comment:
If `getTriggerStyles` isn't used anywhere anymore (which I think it shouldn't be at this point) let's remove it.